Prototyping/Sit/sit.js

```diff
@@ -0,0 +1,692 @@
+                        Overlays.deleteOverlay(overlayPreSitText);
+                    }
+                }
+            },
+            update: function () {
+
+                var now = Date.now();
+
+                if (index >= overlayPreSitLoaded.length) {
+                    return;
+                }
+
+                var cameraRotation = Camera.orientation;
+
+                if (overlayPreSit) {
+
+                    var updateObj = {
+                        rotation: cameraRotation,
+                        position: Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: 0.1, z: -1 })),
+                    }
+
+                    var timePassed = now - overlayPreSitStart;
+                    if (timePassed >= OVERLAY_PRESIT_FRAME_DURATION) {
+                        overlayPreSitStart = now;
+                        index = index + 1;
+                        updateObj.url = overlayPreSitLoaded[index].url;
+                    }
+
+                    Overlays.editOverlay(overlayPreSit, updateObj);
+                    Overlays.editOverlay(overlayPreSitText, {
+                        rotation: cameraRotation,
+                        position: Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: -0.05, z: -1 })),
+                    });
+                }
+            },
+
+        },
+
+        standUp: {
+            create: function () {
+                if (overlayStandUp === null) {
+                    overlayStandUp = Overlays.addOverlay("image3d", {
+                        position: Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: 0, z: -1 })),
+                        rotation: Camera.orientation,
+                        parentId: MyAvatar.sessionUUID,
+                        parentJoint: "Head",
+                        dimensions: { x: 0.2, y: 0.2 },
+                        url: OVERLAY_URL_STANDUP,
+                        ignoreRayIntersection: false,
+                        drawInFront: true,
+                        visible: true,
+                        emissive: true
+                    });
+                }
+            },
+            remove: function () {
+                if (overlayStandUp) {
+                    Overlays.deleteOverlay(overlayStandUp);
+                    overlayStandUp = null;
+                }
+            },
+            update: function () {
+                if (overlayStandUp) {
+                    Overlays.editOverlay(overlayStandUp, { // too inefficient
+                        rotation: Camera.orientation,
+                        position: Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: 0, z: -1 }))
+                    });
+                }
+            }
+        },
+
+        showSittable: function () { // Show overlays when I'm close to the seat
+
+            if (isInEditMode()) {
+                return;
+            }
```

Comment:
Is returning what we want to do here?  I thought maybe we should hide the overlay instead?